src/test/java/seedu/momentum/logic/parser/ProjectBookParserTest.java

```diff
@@ -77,6 +81,15 @@ public void parseCommand_find() throws Exception {
         assertEquals(new FindCommand(new NameContainsKeywordsPredicate(false, keywords)), command);
     }
 
+    @Test
+    public void parseCommand_sort() throws Exception {
+        assertTrue(parser.parseCommand(SortCommand.COMMAND_WORD) instanceof SortCommand);
+        assertTrue(parser.parseCommand(SortCommand.COMMAND_WORD + " 3") instanceof SortCommand);
+        SortCommand command = (SortCommand) parser.parseCommand(SortCommand.COMMAND_WORD + " "
+                + SORT_ORDER + SortCommand.ASCENDING_ORDER + " " + SORT_TYPE + "alpha");
```

Comment:
Might want to replace the `alpha` (and other keywords in other classes) with the public static variables I recommended in `SortCommandParser.java`

Reply:
Fixed!